### PR TITLE
init and crypto

### PR DESCRIPTION
**Pull request by ivy-dong**

### CODE REVIEW

File: crypto/.classpath [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+  <classpathentry kind="src" path="src/main/java" including="**/*.java"/>
```
Comment (genggjh) [inline crypto/.classpath]: src/main/java -> src/

Comment (genggjh) [reply crypto/.classpath]: pls refactor your source code structure, we need not a java subfolder under our java-common project.

src/main/java/com/arxanfintech/common/util/Utils.java -> src/com/arxanfintech/common/util/Utils.java

**Review by genggjh — CHANGES_REQUESTED** [review]

**Review by genggjh — COMMENTED** [review]
Move src/main/java/com/arxanfintech/common/crypto/Hash.java to src/main/java/com/arxanfintech/common/util/Hash.java

File: LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Copyright ArxanFintech Technology Ltd. 2018 All Rights Reserved.
```
Comment (genggjh) [inline LICENSE.txt]: Please follow the project sdk-go-common, you are using a wrong license.txt file.

Comment (ivy-dong) [reply LICENSE.txt]: done.

**Review by genggjh — APPROVED** [review]

**Pull request merged**